packages/core/src/infrastructure/KyRequester.ts

```diff
@@ -0,0 +1,66 @@
+import Ky from 'ky';
```

Comment:
Should this not be located in the browser portion?

Reply:
See previous comment